packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -42,8 +42,7 @@ export const LaunchTriggerAlert = ({
   flow,
   t,
   konnectorRoot,
-  historyAction,
-  withDescription
```

Comment:
This prop is actually needed because we want to control whether the description is shown depending on the context. In particular, we want to see the description in Harvest dialog (where it is set), but not in the paper list in Mes Papiers (where it isn't set).